net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
I had a look at `net/seafile-server` and found that to be overly complicated. Instead I've implemented a solution using `find(1)` such that only files not having the desired permissions and/or owner and group are touched.